orderedsets/__init__.py

```diff
@@ -48,14 +48,15 @@ class OrderedSet(AbstractSet[T]):
     desired.
     """
 
-    def __init__(self, items: Optional[Iterable[T]] = None) -> None:
-        """Create a new :class:`OrderedSet`, optionally initialized with *items*."""
-        if not items:
-            self._dict: Dict[T, None] = {}
-        elif isinstance(items, dict):
-            self._dict = items
+    def __init__(self, *args: Any) -> None:
```

Comment:
A possibly better fix than to do your own varargs processing would be to use a sentinel value:
```
class _NotProvided:
   pass

...
    def __init__(self, items: Union[Iterable[T], Type[_NotProvided]] = _NotProvided -> None:
```

Reply:
Thanks, I change it in [5837e98](https://github.com/matthiasdiener/orderedsets/pull/41/commits/5837e985529769038d2d7b2dff09ee143ae731f4).